src/main/java/Send_Message/RoomMessageHistoryDBAccess.java

```diff
@@ -0,0 +1,21 @@
+package send_message;
+
+
+import java.util.List;
+import java.util.ArrayList;
+public class RoomMessageHistoryDBAccess implements RoomMessageHistoryDBGateway{
+    private List<MessageDBModel> messageList;
+    public RoomMessageHistoryDBAccess() {
+        // Initialize the messageList in the constructor
+        messageList = new ArrayList<>();
+    }
+    @Override
+    public List<MessageDBModel> loadMessages() {
+        return messageList;
```

Comment:
I'm not sure if this is just temp or something, but we need to actually load the messages from db